pmm-app/tests/DbaaS/api/dbaasAPI.js

```diff
@@ -0,0 +1,110 @@
+const assert = require('assert');
+
+const { I } = inject();
+
+module.exports = {
+
+  async apiRegisterCluster(clusterConfiguration, clusterName) {
+    const body = JSON.stringify({ kubernetes_cluster_name: `${clusterName}`, kube_auth: { kubeconfig: `${clusterConfiguration}` } });
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+
+    const response = await I.sendPostRequest('v1/management/DBaaS/Kubernetes/Register', body, headers);
+
+    assert.ok(
+      response.status === 200,
+      `Failed to register cluster with name "${clusterName}". Response message is "${response.data.message}"`,
+    );
+  },
+
+  async apiUnregisterCluster(clusterName, forceBoolean = false) {
+    const body = JSON.stringify({ kubernetes_cluster_name: `${clusterName}`, force: forceBoolean });
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+
+    const response = await I.sendPostRequest('v1/management/DBaaS/Kubernetes/Unregister', body, headers);
+
+    assert.ok(
+      response.status === 200,
+      `Failed to unregister cluster with name "${clusterName}". Response message is "${response.data.message}"`,
+    );
+  },
+
+  async apiDeleteXtraDBCluster(dbClusterName, clusterName) {
+    const body = JSON.stringify({ kubernetes_cluster_name: `${clusterName}`, name: dbClusterName });
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+
+    const response = await I.sendPostRequest('v1/management/DBaaS/XtraDBCluster/Delete', body, headers);
+
+    assert.ok(
+      response.status === 200,
+      `Failed to delete pxc cluster with name "${dbClusterName}". Response message is "${response.data.message}"`,
+    );
+  },
+
+  async apiCheckRegisteredClusterExist(clusterName) {
+    const body = JSON.stringify({});
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+
+    const response = await I.sendPostRequest('v1/management/DBaaS/Kubernetes/List', body, headers);
+
+    if (typeof response.data.kubernetes_clusters !== 'undefined') {
+      const cluster = response.data.kubernetes_clusters.find(
+        (o) => o.kubernetes_cluster_name === clusterName,
+      );
+
+      return typeof cluster !== 'undefined';
```

Comment:
```suggestion
      return response.data.kubernetes_clusters.includes(clusterName);
```

Reply:
That'd be much shorter )

Reply:
I am not sure, because it is an array of objects and I am first searching for the object with a matching cluster name.

Reply:
It will, it does exactly the same. Pls try, since we tend to have a lot of copy/paste, and that sets out a good example of how to solve it.

Reply:
I have tried it..it doesn't work and I am not sure if I am confused here 
```
const kubernetes_clusters = [
  {
    kubernetes_cluster_name: 'Cluster1',
    operators: { xtradb: [Object], psmdb: [Object] },
    status: 'KUBERNETES_CLUSTER_STATUS_OK'
  },
  {
    kubernetes_cluster_name: 'Testing',
    operators: { xtradb: [Object], psmdb: [Object] },
    status: 'KUBERNETES_CLUSTER_STATUS_OK'
  },
  {
    kubernetes_cluster_name: 'Kubernetes_Testing_Cluster_Minikube',
    operators: { xtradb: [Object], psmdb: [Object] },
    status: 'KUBERNETES_CLUSTER_STATUS_OK'
  }
];
```

That's the structure of kubernetes_clusters and if the array contains strings/numbers then I think array.includes would work just fine, but in this case, we need to filter by specific property of the object, hence I am using find and checking property `kubernetes_cluster_name`

Reply:
Correct, that wouldn't work in your case ))